keps/prod-readiness/sig-node/1797.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 1797
```

Comment:
actually - can you please add a "beta" entry too (migrating the previous prr-approver field to here) [same as you did with stable]